TwitchPlaysAssembly/Src/Votes.cs

```diff
@@ -0,0 +1,145 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Timers;
+
+public static class Votes
+{
+	public static bool _going = false;
+	public static bool _global = false;
```

Comment:
Why do you have `_going` and `_global`? It seems like they track a distinction between a globally ran vote or a bomb vote but that distinction never actually effects anything. They could just be combined into one variable instead.

Reply:
When there’s a state change or a new factory bomb Clear() gets called which cancels the current vote in case it’s a bomb vote. If it’s a global vote it shouldn’t do anything. That’s why I keep different variables for them